src/v4/analytics.ts

```diff
@@ -0,0 +1,76 @@
+import { Client } from '../client'
+import { BaseError } from '../errors'
+import { UriHelper } from '../uri-helper'
+import { ThBaseHandler } from '../base'
+
+export interface AnalyticsOptions {
+  user?: string
+  base?: string
+}
+
+export interface AnalyticsGetRevenueAverageQuery {
```

Comment:
these 4 filters are the base ones. They're all the same for all the v4 analytics endpoints
Means you can rename the interface to AnalyticsQuery or AnalyticsBaseQuery